### PR TITLE
cpu/cc26x0: implement i2c master

### PR DESCRIPTION
**Pull request by benemorius**
This adds I2C master support to the CC26x0 port.
- tested with CC2650 and SI7006
- slave mode not implemented
- `i2c_write_regs` and `i2c_read_regs` not implemented
- DMA not implemented

### CODE REVIEW

File: cpu/cc26x0/include/cc26x0_ioc.h [hunk 1]
```diff
@@ -128,6 +128,104 @@ typedef struct {
...
+
+//*****************************************************************************
+//
+// Values that can be used to set the IO Mode of an IO
+//
+//*****************************************************************************
+#define IOC_IOMODE_NORMAL       0x00000000  // Normal Input/Output
+#define IOC_IOMODE_INV          0x01000000  // Inverted Input/Output
+#define IOC_IOMODE_OPEN_DRAIN_NORMAL \
+0x04000000  // Open Drain, Normal Input/Output
```
Comment (A-Paul) [inline cpu/cc26x0/include/cc26x0_ioc.h]: Please indent warped lines.

**Comment by A-Paul** [comment]
Hi @benemorius, thanks for your contribution.

On first sight, please replace line-comments ``//`` with block-comments ``/* */``.

**Comment by shaolinkhoa** [comment]
Hi,
Are there any updates or example project using I2c in Riot os for cc2650 ?
thank you

**Comment by PeterKietzmann** [comment]
Did you try [tests/driver_si70xx](https://github.com/RIOT-OS/RIOT/tree/master/tests/driver_si70xx)? See @benemorius PR description above.

**Comment by PeterKietzmann** [comment]
@benemorius are you still around and willing to contribute?

**Comment by A-Paul** [comment]
As there is no reaction by the original author, I close this PR.
I'm going to adopt the branch from @benemorius and try to continue the implementation in a new PR.

**Pull request closed without merging**